index.html

```diff
@@ -53,8 +52,7 @@
     <div class="tab-content-item show" id="contact-content"> This is for Contact</div>
   </main>
   <footer>
-    <p>Author: Hege Refsnes</p>
-    <p><a href="mailto:hege@example.com">hege@example.com</a></p>
+    Copyright © 2020 - Flavia Spataru
```

Comment:
cum altfel ai putea scrie caracterul special de Copyright, astfel incat orice browser in orice Unicode sa il inteleaga?